public/app/plugins/datasource/grafana-azure-monitor-datasource/plugin.json

```diff
@@ -183,108 +183,6 @@
-        }
-      },
-      "headers": [{ "name": "x-ms-app", "content": "Grafana" }]
-    },
-    {
-      "path": "loganalytics-resourcepickerdata-gov",
-      "method": "POST",
-      "url": "https://management.usgovcloudapi.net",
-      "authType": "azure",
-      "tokenAuth": {
-        "scopes": ["https://management.usgovcloudapi.net/.default"],
-        "params": {
-          "azure_auth_type": "{{.JsonData.azureAuthType | orEmpty}}",
-          "azure_cloud": "AzureUSGovernment",
-          "tenant_id": "{{.JsonData.logAnalyticsTenantId | orEmpty}}",
-          "client_id": "{{.JsonData.logAnalyticsClientId | orEmpty}}",
-          "client_secret": "{{.SecureJsonData.logAnalyticsClientSecret | orEmpty}}"
-        }
-      },
-      "headers": [{ "name": "x-ms-app", "content": "Grafana" }]
-    },
-    {
-      "path": "workspacesloganalytics",
-      "method": "GET",
-      "url": "https://management.azure.com",
-      "authType": "azure",
-      "tokenAuth": {
-        "scopes": ["https://management.azure.com/.default"],
-        "params": {
-          "azure_auth_type": "{{.JsonData.azureAuthType | orEmpty}}",
-          "azure_cloud": "AzureCloud",
-          "tenant_id": "{{.JsonData.logAnalyticsTenantId | orEmpty}}",
-          "client_id": "{{.JsonData.logAnalyticsClientId | orEmpty}}",
-          "client_secret": "{{.SecureJsonData.logAnalyticsClientSecret | orEmpty}}"
-        }
-      },
-      "headers": [{ "name": "x-ms-app", "content": "Grafana" }]
-    },
-    {
-      "path": "chinaworkspacesloganalytics",
-      "method": "GET",
-      "url": "https://management.chinacloudapi.cn",
-      "authType": "azure",
-      "tokenAuth": {
-        "scopes": ["https://management.chinacloudapi.cn/.default"],
-        "params": {
-          "azure_auth_type": "{{.JsonData.azureAuthType | orEmpty}}",
-          "azure_cloud": "AzureChinaCloud",
-          "tenant_id": "{{.JsonData.logAnalyticsTenantId | orEmpty}}",
-          "client_id": "{{.JsonData.logAnalyticsClientId | orEmpty}}",
-          "client_secret": "{{.SecureJsonData.logAnalyticsClientSecret | orEmpty}}"
-        }
-      },
-      "headers": [{ "name": "x-ms-app", "content": "Grafana" }]
-    },
-    {
-      "path": "govworkspacesloganalytics",
-      "method": "GET",
-      "url": "https://management.usgovcloudapi.net",
-      "authType": "azure",
-      "tokenAuth": {
-        "scopes": ["https://management.usgovcloudapi.net/.default"],
-        "params": {
-          "azure_auth_type": "{{.JsonData.azureAuthType | orEmpty}}",
-          "azure_cloud": "AzureUSGovernment",
-          "tenant_id": "{{.JsonData.logAnalyticsTenantId | orEmpty}}",
-          "client_id": "{{.JsonData.logAnalyticsClientId | orEmpty}}",
-          "client_secret": "{{.SecureJsonData.logAnalyticsClientSecret | orEmpty}}"
-        }
-      },
-      "headers": [{ "name": "x-ms-app", "content": "Grafana" }]
-    },
     {
       "path": "loganalyticsazure",
```

Comment:
Is it only the scope that is different between this route and azuremonitor? I wonder if the "https://management.azure.com/.default" scope could also be used for loganalyticsazure. In that case we might be able to get rid of this route. Might be worth testing.